extensions/amp-social-share/1.0/amp-social-share.js

```diff
@@ -158,6 +158,8 @@ class AmpSocialShare extends PreactBaseElement {
 /** @override */
 AmpSocialShare['Component'] = SocialShare;
 
+AmpSocialShare['passthroughNonEmpty'] = true;
```

Comment:
Is this going to be defined in `PreactBaseElement`? Does it do anything now?

Reply:
I added this in a separate PR here: https://github.com/ampproject/amphtml/pull/29367.  Wasn't really sure if I should document it somewhere.  I suppose the code base is the documentation for now.

Basically the behavior is the same as 'passthrough' except if there are no children it will return `null` instead of a `slot`.  And it's also recommended to use mutation observer when using this new `passthroughNonEmpty`.